app/controllers/opendoors/external_authors_controller.rb

```diff
@@ -2,7 +2,7 @@ class Opendoors::ExternalAuthorsController < ApplicationController
 
   before_action :users_only
   before_action :opendoors_only
-  before_action :load_external_author, only: [:show, :edit, :update, :forward]
+  before_action :load_external_author, only: [:show, :forward]
```

Comment:
Technically this was already part of the initial 7.1 upgrade (`raise_on_missing_callback_actions`), but I later noticed that this controller wasn't covered by tests, so this instance was not caught.